frontend/src/app/components/standup-detail/standup-detail.component.html

```diff
@@ -24,19 +24,19 @@ <h2>Blockers: {{ progress.blocker }}</h2>
 
     </div>
   </div><br />
-    <hr>
+  <hr>
   <div style='max-width: 75%'>
     <div *ngIf="!selectedProgress">
       test
     </div>
 
     <div *ngIf="selectedProgress">
-      Accomplished <input [(ngModel)]="selectedProgress.accomplished"><br />
-      Working On <input [(ngModel)]="selectedProgress.working_on"><br />
-      Blocker <input [(ngModel)]="selectedProgress.blocker"><br />
-    <button class="button is-danger is-large" (click)="createProgress()">Create New</button>
+      Accomplished <input [(ngModel)]="selectedProgress.accomplished" class="input"><br />
+      Working On <input [(ngModel)]="selectedProgress.working_on" class="input"><br />
+      Blocker <input [(ngModel)]="selectedProgress.blocker" class="input"><br />
+      <button class="button is-danger is-large" (click)="createProgress()">Create New</button>
       <button class="button is-danger is-large" *ngIf="selectedProgress.id" (click)="updateProgress()">Update</button>
       <button class="button is-danger is-large" *ngIf="selectedProgress.id" (click)="deleteProgress()">Delete</button>
     </div>
   </div>
-</section>
+</section>
```

Comment:
Looks like there was a trailing new line that you removed. We want to keep those